src/devices/cpu/tms34010/34010ops.hxx

```diff
@@ -2395,7 +2399,40 @@ void tms340x0_device::rpix_b(uint16_t op)
 void tms340x0_device::setcdp(uint16_t op)
 {
 	if (!m_is_34020) { unimpl(op); return; }
-	logerror("020:setcdp\n");
+	off_t dptch = DPTCH();
+
+	// logerror("020:setcdp: PC=%x, DPTCH()=%x\n", m_pc, dptch);
+
+	// Check whether we're dealing with an even number
+	if ((dptch & 1) == 0) {
+		switch(population_count_32(dptch))
+		{
```

Comment:
Be consistent with code style - if the file uses K&R style, don't use Allman, and vice versa.